README.md

```diff
@@ -58,7 +60,46 @@ Experimenter is a platform for managing experiments in [Mozilla Firefox](https:/
 
         https://localhost/
 
-Done!
+#### Semi Dockerized Setup (continuation from General Setup 1-7)
+
+Node ^14.0.0 is required
```

Comment:
I actually ran into a problem with this when setting up a yarn workspace in a new project since `nimbus-shared` uses Node 14, this might be better up towards the top so anyone running the project (even "fully dockerized") knows to use Node 14.